src/contracts/LiquidityReserve.sol

```diff
@@ -154,15 +162,18 @@ contract LiquidityReserve is ERC20, Ownable {
         IERC20(stakingToken).safeTransfer(msg.sender, amountToWithdraw);
     }
 
-    //TODO: clean up natspec
     /**
         @notice allow instant unstake their stakingToken for a fee paid to the liquidity providers
-        @param _amount uint
-        @param _recipient address
+        @param _amount uint - amount of tokens to instantly unstake
+        @param _recipient address - address to send staking tokens to
      */
-    function instantUnstake(uint256 _amount, address _recipient) external {
+    function instantUnstake(uint256 _amount, address _recipient)
+        external
+        onlyStakingContract
```

Comment:
This ended up needing to be only staking due to only allowing the full amount to be unstaked.  I had this in the previous pr, but it must have gotten reverted somehow

Reply:
thats fine, i think it also makes the flow more obvious to the user who wont try and call this version